samples/openthread/ncp/README.rst

```diff
@@ -61,6 +83,41 @@ Building and running
 
 .. include:: /includes/build_and_run.txt
 
+.. _ot_ncp_sample_features_enabling:
+
+Enabling additional features
+============================
+
+When building the sample, you can enable :ref:`additional sample features <ot_ncp_sample_features>`.
+To do this, run the build command with one additional parameter that defines the configuration files that you want to use.
+For example, to enable vendor hooks on the nRF52840 DK, run the following command:
+
+.. code-block:: console
+
+   west build -b nrf52840dk_nrf52840 -- -DCONF_FILE="prj.conf overlay-vendor_hook.conf"
```

Comment:
```suggestion
When building the sample, you can enable :ref:`additional sample features <ot_ncp_sample_features>`.
To do this, modify :makevar:`OVERLAY_CONFIG`.
For example, to enable vendor hooks, set :file:`overlay-vendor_hook.conf`
See :ref:`cmake_options` for instructions on how to add this option.
```

Reply:
Done it. @b-gent do you think it needs tech writer re-review?

Reply:
This look good.